docs/pages/build-reference/npm-hooks.mdx

```diff
@@ -10,6 +10,7 @@ import { CODE } from '~/ui/components/Text';
 EAS Build lifecycle npm hooks allows you to customize your build process by running scripts before or after the build process.
 
 > For better understanding, see the [Android build process](/build-reference/android-builds/) and the [iOS build process](/build-reference/ios-builds/).
+> **warning** The lifecycle hooks are not executed by build process in [custom builds](/custom-builds/get-started/). They need to be manually extracted to build steps called when running the build.
```

Comment:
```suggestion

> **warning** The lifecycle hooks are not executed by build process in [custom builds](/custom-builds/get-started/). They need to be manually extracted to build steps called when running the build.
```

Quote blocks need to be separated by new line, to be rendered as two components.

![Screenshot 2024-08-30 at 14 32 35](https://github.com/user-attachments/assets/ba7f1e99-18e4-4949-9c61-78426f390095)

![Screenshot 2024-08-30 at 14 34 27](https://github.com/user-attachments/assets/7ed8799d-ec2b-473a-be7d-9cb1e1e406e2)